### PR TITLE
Add getInsertGenerator, getUpdateGenerator to Platform interface 

### PR DESCRIPTION
**Pull request by debop**
For Customizing insert, update sql statement (ex. Apache Phoenix support `UPSERT` only)

### CODE REVIEW

File: requery/src/main/java/io/requery/sql/platform/Generic.java [hunk 1]
```diff
@@ -19,17 +19,9 @@
...
-import io.requery.sql.UserVersionColumnDefinition;
-import io.requery.sql.VersionColumnDefinition;
-import io.requery.sql.gen.Generator;
-import io.requery.sql.gen.LimitGenerator;
-import io.requery.sql.gen.OffsetFetchGenerator;
-import io.requery.sql.gen.OrderByGenerator;
-import io.requery.sql.gen.UpsertMergeGenerator;
+import io.requery.query.element.QueryElement;
+import io.requery.sql.*;
+import io.requery.sql.gen.*;
```
Comment (npurushe) [inline requery/src/main/java/io/requery/sql/platform/Generic.java]: Please avoid changing the imports to wildcard, (avoid wildcard imports)

Comment (debop) [reply requery/src/main/java/io/requery/sql/platform/Generic.java]: Sorry, It was by IntelliJ Settings, reformat at home

**Comment by debop** [comment]
Any problem?

**Comment by mcun0s** [comment]
GOOOOOOOOOOOOOOOD NEWS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by mcun0s** [comment]
@npurushe when do you expect created next release for this pool request? It would be good if we can know that for our planning. many thanks!

**Pull request merged**